server/navigation/_beginner_resources.py

```diff
@@ -14,11 +14,12 @@ def get_filepath_dict():
             "New to RPGs?",
             '/newplayer',
             ('/rules/overview', '/races', '/classes', '/files'),
-            ('/rules/combat', '/files', '/rules/glossary')
+            ('/rules/checks', '/rules/combat', '/files', '/rules/glossary')
```

Comment:
Will this be another document that doesn't yet exist? i.e. breaking basic rules up a little further?